### PR TITLE
QueryStore_Additional_Data

### PR DESCRIPTION
**Pull request by jagriffin5**
Added new columns and table for QS modifications.
Added columns force_failure_count, last_force_failure_reason_desc and plan_forcing_type_desc to table sys.query_store_plan. Also, added new table tbl_dm_db_tuning_recommendations to collect auto tune recommendations from new query in SQLLogScout for sys.dm_db_tuning_recommendations.

### CODE REVIEW

File: sqlnexus/TextRowsets.xml [hunk 1]
```diff
@@ -2235,6 +2235,28 @@
...
       <Column name="query_plan_hash" type="RowsetImportEngine.VarBinaryColumn" length="8" />
       <Column name="is_forced_plan" type="RowsetImportEngine.IntColumn" />
+      <!-- 620 JAMGRIF 022024 -->
+      <Column name="force_failure_count" type="RowsetImportEngine.BigIntColumn" />
+      <Column name="last_force_failure_reason_desc"  type="RowsetImportEngine.VarCharColumn" length="256" />
+      <!-- 620 JAMGRIF 022024 2017+ only-->
+      <Column name="plan_forcing_type_desc"  type="RowsetImportEngine.VarCharColumn" length="120" />
+    </Rowset>
+
+    <!-- 620 JAMGRIF 022024 -->
```
Comment (PiJoCoder) [inline sqlnexus/TextRowsets.xml:2245]:
```suggestion
```

Comment (PiJoCoder) [reply sqlnexus/TextRowsets.xml:2245]: Basically, rebuild your solution, do another commit and push it.

**Review by PiJoCoder — CHANGES_REQUESTED** [review]
Looks good - tested the functionality and it works great. 
Just recompile the solution and let the TextRowset.xml file be automatically generated in the \SQLNexus folder

File: sqlnexus/TextRowsets.xml (hunk 1 above)
Comment (jagriffin5) [reply sqlnexus/TextRowsets.xml]: Done.

**Pull request merged**